.travis.yml

```diff
@@ -3,7 +3,7 @@ git:
   quiet: true
 
 language: c
-dist: trusty
+dist: bionic
```

Comment:
Curious as to why since we have CircleCI. Even on 18.06.

Given that CircleCI uses a more modern distro, could be useful to see if it fails on an older one.

Reply:
Not everyone is using CircleCI, when there is exist an alternative and someone can be used to it. Why don't use it, right? Looking at [Build system - Installation](https://openwrt.org/docs/guide-developer/build-system/install-buildsystem), there is mention current Ubuntu 18.04 LTS. I don't think we want to use GNU/Linux distribution with ended standard support. I was thinking about Ubuntu 16.04 LTS, but let's use the newest LTS.